draft-api/src/main/scala/no/ndla/draftapi/service/ConverterService.scala

```diff
@@ -249,8 +244,8 @@ trait ConverterService {
       common.ArticleContent(removeUnknownEmbedTagAttribute(articleContent.content), articleContent.language)
     }
 
-    private def toDomainTag(tag: Seq[String], language: String): Option[common.Tag] =
-      if (tag.nonEmpty) Some(common.Tag(tag, language)) else None
+    private def toDomainTag(tag: Option[Seq[String]], language: String): Option[common.Tag] =
+      if (tag.getOrElse(Seq.empty).nonEmpty) Some(common.Tag(tag.get, language)) else None
```

Comment:
Kanskje vi kan skrive denne uten `.get`'en?
```suggestion
      tag.collect { case list if list.nonEmpty => common.Tag(list, language) }
```

Evt om collect blir rart
```suggestion
      tag.flatMap {
        case list if list.nonEmpty => Some(common.Tag(list, language))
        case _                     => None
      }
```